src/sections/Meshery/How-meshery-works/howitworks.style.js

```diff
@@ -71,6 +71,7 @@ h2 {
   }
   
   .diagram {
+    margin-bottom:150px;
```

Comment:
why extra margin?

Reply:
@sudhanshutech Left div will go to the end of the parent div, compared to the right div, which looks bad. So, I added a margin there.

Reply:
![Screenshot 2024-12-25 at 11 44 14 AM](https://github.com/user-attachments/assets/d69c56d5-2e66-4126-9209-c8557e3021ed)

Reply:
You can see the left div reaching the end.

Reply:
And this is after adding margin
![Screenshot 2024-12-25 at 11 48 31 AM](https://github.com/user-attachments/assets/b1d11d26-26eb-4f54-8935-0c468a5948e2)